examples/book-with-adult-and-infant.py

```diff
@@ -0,0 +1,111 @@
+from datetime import date
+
+from duffel_api import Duffel
+
+
+if __name__ == "__main__":
+    print("Duffel Flights API - book with adult and infant example")
+    client = Duffel()
+    departure_date = date.today().replace(date.today().year + 1)
+    slices = [
+        {
+            "origin": "LHR",
+            "destination": "STN",
+            "departure_date": departure_date.strftime("%Y-%m-%d"),
+        },
+    ]
+    offer_request = (
+        client.offer_requests.create()
+        .passengers(
+            [{"type": "adult"}, {"age": 1}, {"age": (date.today().year - 2003)}]
+        )
+        .slices(slices)
+        .execute()
+    )
+
+    print("Created offer request: %s" % (offer_request.id))
+
+    offers = client.offers.list(offer_request.id)
+    offers_list = list(enumerate(offers))
+
+    print("Got %d offers" % len(offers_list))
```

Comment:
```suggestion
    print(f"Got {len(offers_list)} offers")
```

There's some other places we could use f-strings too.